src/app/helptext/storage/volumes/datasets/dataset-form.ts

```diff
@@ -40,6 +40,18 @@ export default {
   dataset_form_refquota_tooltip: T('<i>0</i> disables quotas. Specify a maximum allowed\
  space for this dataset.'),
 
+  acl_type_change_warning: T('Changes to ACL type affect how on-disk ZFS ACL is \
+ written and read.\n\nWhen the ACL type is changed from POSIX to NFSv4, \
+ no migration is performed for default and access ACLs encoded in the \
+ posix1e acl extended attributes to native ZFS ACLs.\n\nWhen ACL type is \
+ changed from NFSv4 to POSIX, native ZFS ACLs are not converted to \
+ posix1e extended attributes, but the native ACL will be used internally \
+ by ZFS for access checks.\n\nThis means that the user must manually set \
+ new ACLs recursively on the dataset after ACL type changes in order to \
+ avoid unexpected permissions behavior.\n\nThis action will be destructive, \
+ and so it is advised to take a ZFS snapshot of the dataset prior to ACL \
+ type changes and permissions modifications.'),
```

Comment:
Sorry, can we keep the line break, but remove the empty line at least on the first two.
We need to indicate to the user that there is more text there and that he can scroll.